WebApp/Scripts/MarkupPanel.js

```diff
@@ -119,6 +125,43 @@ var GPV = (function (gpv) {
       }
     }).on("blur", updateMarkupGroupTitle);
 
+    // date 05-08-2017
+    var $tboMarkupDetails = $("#tboMarkupDetails").on("keydown", function (e) {
```

Comment:
Change 
```tboMarkupDetails``` to ```tblMarkupDetails```

Reply:
and also put comments for functionalities

Reply:
done